Gems/PhysX/Code/Tests/PhysXColliderComponentModeTests.cpp

```diff
@@ -6,48 +6,71 @@
  *
  */
 
-#include <PhysXColliderComponentModeTests.h>
-
+#include <TestColliderComponent.h>
+
+#include <AzManipulatorTestFramework/IndirectManipulatorViewportInteraction.h>
+#include <AzManipulatorTestFramework/AzManipulatorTestFrameworkTestHelpers.h>
+#include <AzManipulatorTestFramework/AzManipulatorTestFrameworkUtils.h>
+#include <AZTestShared/Math/MathTestHelpers.h>
+#include <AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.h>
+#include <AzToolsFramework/ViewportSelection/EditorInteractionSystemViewportSelectionRequestBus.h>
+#include <AzToolsFramework/ViewportSelection/EditorDefaultSelection.h>
+#include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+#include <AzToolsFramework/Viewport/ViewportSettings.h>
+#include <AzToolsFramework/ViewportUi/ViewportUiManager.h>
+#include <AzToolsFramework/ToolsComponents/EditorNonUniformScaleComponent.h>
 #include <EditorColliderComponent.h>
+#include <EditorRigidBodyComponent.h>
 #include <EditorStaticRigidBodyComponent.h>
-#include <AzToolsFramework/ToolsComponents/EditorNonUniformScaleComponent.h>
 
 namespace UnitTest
 {
-    AZ::Entity* PhysXColliderComponentModeTest::CreateColliderComponent()
+    class PhysXColliderComponentModeTest
+        : public ToolsApplicationFixture<false>
     {
-        AZ::Entity* entity = nullptr;
-        AZ::EntityId entityId = CreateDefaultEditorEntity("ComponentModeEntity", &entity);
+    protected:
+        using EntityPtr = AZ::Entity*;
 
-        entity->Deactivate();
+        AZ::ComponentId m_colliderComponentId;
 
-        // Add placeholder component which implements component mode.
-        auto colliderComponent = entity->CreateComponent<TestColliderComponentMode>();
+        EntityPtr CreateColliderComponent()
+        {
+            AZ::Entity* entity = nullptr;
```

Comment:
nit: Especially when we don't use it 😅

Reply:
`EntityPtr` was used as returning type of `CreateColliderComponent`, It was there like that before, but I agree it's better to remove it. I'll remove it.

Reply:
Excellent thanks! 🙂